smart-locc.ino

```diff
@@ -37,22 +49,45 @@ void setup() {
     Serial.println("Booted up.");
 }
 
-void loop() {
+void loop()
+{
+    if (!client.connected())
+    {
+        Serial.print("Reconnecting to server...");
+        bool connected = client.connect(CLIENT_ID, USER_TOKEN, PASS);
+        Serial.println(connected ? "success" : "failure");
+    }
+    else
+    {
+        client.loop();
+    }
+    delay(2000);
+
+    // -----------------------------------------------------
+
     stateManager.setCurrentTime(millis());
     const DeviceState currentState = stateManager.getState();
-    if (currentState != lastState) {
+    if (currentState != lastState)
+    {
         lastState = currentState;
         Serial.print("STATE -> ");
         Serial.println(currentState);
 
         // TODO: update lock state, display info, etc
         // ...
         //
-
+        if (currentState == DeviceState::AUTHORIZED || currentState == DeviceState::AUTH_FAILURE)
+        {
+            char data[250];
```

Comment:
250 is a lot of bytes, better write it like
```cpp
const char* data = currentState == DeviceState::AUTHORIZED ? VARIABLE_LABEL_SUCCESSFUL : VARIABLE_LABEL_UNSUCCESSFUL
```

Reply:
Or alternatively, do not declare those 2 constants using `#define` but instead have them be `const char*` in `Config.h`

Reply:
Updated